cf/api/resources/service_keys_test.go

```diff
@@ -0,0 +1,72 @@
+package resources_test
+
+import (
+	"encoding/json"
+
+	. "github.com/cloudfoundry/cli/cf/api/resources"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ServiceKeyResource", func() {
+	var resource ServiceKeyResource
+
+	BeforeEach(func() {
+		err := json.Unmarshal([]byte(`
+    {
+      "metadata": {
+        "guid": "fake-service-key-guid",
+        "url": "/v2/service_keys/fake-guid",
+        "created_at": "2015-01-13T18:52:08+00:00",
+        "updated_at": null
+      },
+      "entity": {
+        "name": "fake-service-key-name",
+        "service_instance_guid":"fake-service-instance-guid",
+        "service_instance_url":"http://fake/service/instance/url",
+        "credentials": {
+          "username": "fake-username",
+          "password": "fake-password",
+          "host": "fake-host",
+          "port": 3306,
+          "database": "fake-db-name",
+          "uri": "mysql://fake-user:fake-password@fake-host:3306/fake-db-name"
+        }
+      }
+    }`), &resource)
+
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Context("Brokers supporting service keys", func() {
```

Comment:
Do we need this context? I ask because, it begs the question whether there would be a mirror context: `Brokers NOT supporting service keys`. Not a huge issue but typically if there is no need for context just inline all of the `It` tests in the `Describe`... Of course, if you are following a pattern then cool.